src/createAdder.js

```diff
@@ -22,7 +22,13 @@
  * @return {number}
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let elementsSum = initialValue;
+
+  return function(...numbers) {
+    const sumArgs = [...numbers].reduce((a, b) => a + b, 0);
```

Comment:
зачем отдельная переменная?
```suggestion
    elementsSum += [...numbers].reduce((a, b) => a + b, 0);
```